tools/docker/integration.Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM node:10.15 
+
+# Install docker and docker compose
+RUN apt-get update \
+    #
+    # Install Docker CE CLI
+    && apt-get install -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common lsb-release \
+    && curl -fsSL https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]')/gpg | apt-key add - 2>/dev/null \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y docker-ce-cli \
+    #
+    # Install Docker Compose
+    && curl -sSL "https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose \
+    && chmod +x /usr/local/bin/docker-compose \
+    #
+    # Install jq
+    && apt-get -y install jq
+    
+# Install dependencies needed to run cypress with chrome
+RUN apt-get update && apt-get install -y xvfb libgtk-3-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 fonts-liberation libappindicator3-1 xdg-utils
```

Comment:
Could this be merged into the above apt-get steps as part of an &&? multiple apt-get updates in a single docker file seems like it could cause problems.

Reply:
This file has been a victim of copy-paste

Reply:
8a4e092ecb27cf42cf91da8a4fa931f4d357a66a